lib/gmath/la.c

```diff
@@ -1326,7 +1326,7 @@ double G_vector_norm1(vec_struct * vc)
 
     if (!vc->is_init) {
         G_warning(_("Matrix is not initialised"));
-        return 0.0 / 0.0;       /* NaN */
+        return NAN;       /* NaN */
```

Comment:
You may also remove the comment `/* NaN */` and similar comments in other cases as the code is self-explanatory.

Reply:
Sure, done in abec3809a8879956f9bac351f3083389423dd091